metabase/metabase
# Add `in` and `notIn` custom expression functions

Epic https://github.com/metabase/metabase/issues/51134

How to verify:
- New -> Question -> People
- Filter -> Custom expression -> `in([Source], "Google", "Twitter")` -> Done
- Visualize and make sure that results are correct
- Notebook -> Click on the filter -> Make sure that the filter widget recognized the new expression shape
- Update the filter, e.g. add `Organic` -> Click Update filter
- Click on the new filter -> Back -> Custom expression -> Make sure the widget used `in` for multi-arg expression

<img width="623" alt="Screenshot 2024-12-11 at 23 07 17" src="https://github.com/user-attachments/assets/4a1648ea-acee-4974-bf09-2b196ee3425a" />
